validity/Cargo.toml

```diff
@@ -63,9 +64,3 @@ strum_macros.workspace = true
 
 [build-dependencies]
 op-succinct-build-utils.workspace = true
-
-[features]
-# Building without --features will use `SingleChainOPSuccinctHost` with Ethereum DA.
-default = []
-# Feature flag for Celestia DA support.
-celestia = ["op-succinct-host-utils/celestia"]
```

Comment:
If `initialize_host` has the Celestia host gated behind a feature flag, how will this be enabled?

Reply:
Just running `cargo run --bin validity --release --features celestia` enables it, no need to declare features here.

Added logging to visualize it more clearly.
```
2025-05-12T17:27:57.571267Z  INFO Initializing host with Celestia DA
```

Reply:
But for better visibility, I'll add feature flags.